### PR TITLE
Add solution for #5 HTML & CSS Practice

### PR DESCRIPTION
**Pull request by Ant-C-tech**
###HTML & CSS Practice

[Demo](https://ant-c-tech.github.io/-5-HTML-CSS-Practice/) |
[Code base](https://github.com/Ant-C-tech/-5-HTML-CSS-Practice)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/Ant-C-tech/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        <img class="intro__img intro__imgAnimate" src="./img/cat-smile.jpg" alt="cat">
+        <p class="intro__footer">Icon by <a class="intro__footerLink"
+                href="https://www.flaticon.com/free-icon/maps_1673429?term=map&page=2&position=8&related_item_id=1673429"
+                target="_blank">Flaticon</a>
+        </p>
+    </div>
+
+</body>
+
+</html>
```
Comment (lenadgit) [inline submissions/Ant-C-tech/html-css-popup/index.html:162]: could you please add an empty line at the end of the file?  If you see this symbol it means that an empty line is needed
empty line

**Review by lenadgit — COMMENTED** [review]
Really cool work! one of the best completing of this task

File: submissions/Ant-C-tech/html-css-popup/index.html (hunk 1 above)
Comment (lenadgit) [inline submissions/Ant-C-tech/html-css-popup/index.html:162]: could you please add an empty line at the end of the file?  If you see this symbol it means that an empty line is needed
empty line

**Review by lenadgit — CHANGES_REQUESTED** [review]

**Comment by Ant-C-tech** [comment]
Please, review my solution.

**Review by lenadgit — APPROVED** [review]

**Comment by Ant-C-tech** [comment]
Thank you for merge and help with normalize css!

**Pull request merged**